### PR TITLE
Added files via upload

### PR DESCRIPTION
**Pull request by nishtha3006**
fixed #12 sign up html .
fixed #13 login html.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
+body {
+background: url("http://wp.givingtuesday.ca/app/uploads/2014/11/GivingTuesday_BlogHeader.jpg") no-repeat center center fixed;
```
Comment (hkasera) [inline style.css]: Format this file. It still uses the http url instead the image you added. Please check your changes thoroughly before committing.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+font-size: 12px;
+height: 50px;
+line-height: 50px;
+margin: 50px 150px;
+text-align: center;
+text-transform: uppercase;
+width: 50px;
+}
+
+.SignUp {
```
Comment (hkasera) [inline style.css]: The naming convention of classes is not consistent. In some places you have used lower case for class names, whereas here you have used camel case convention. Stick to one.

**Comment by hkasera** [comment]
Don't create separate pull requests like these. You can edit the previous ones itself. If you need help in that, please let me know!

**Comment by nishtha3006** [comment]
ohk sry for inconvenience.

**Pull request closed without merging**